### ISSUE TITLE
TFLite model exported from Keras cannot be executed on Android

### ISSUE THREAD

**Issue by PiotrSwietek** [tensorflow/tensorflow#29041]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows/Android
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: Huawei P10
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.13.1
- **Python version**: 3.6
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**: Geforce Gtx1080 8Gb
- **Exact command to reproduce**:

### Describe the problem
I have exported a model from Keras using following code:
```

converter = lite.TFLiteConverter.from_keras_model_file(weights_path_2)
tfmodel = converter.convert()
open ("model.tflite" , "wb") .write(tfmodel)

```

Export and import into Android works fine, executing however throws following error:
```

java.lang.IllegalStateException: Internal error: Unexpected failure when preparing tensor allocations: tensorflow/lite/kernels/fully_connected.cc:112 input_size != batch_size * filter->dims->data[1] (4096 != 0)Node number 7 (FULLY_CONNECTED) failed to prepare.

```
I also tried exporting the model using toco, but the same issue happens.
The model I used here can be found here https://github.com/krasserm/face-recognition/blob/master/model.py
Any idea on how to fix this? 
### Describe expected behaviour

If the model can be exported without problems I should be able to executed it too.

**Comment by jdduke** [comment]
Are you setting the size of your inputs? If you're using the Java bindings, how are you invoking the model? What shapes are you using?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by Taichi-Pink** [comment]
I met a similar error. Did you fix this error? @PiotrSwietek
